src/main/kotlin/com/github/patou/gitmoji/GitMojiConfig.kt

```diff
@@ -26,7 +26,10 @@ class GitMojiConfig(private val project: Project) : SearchableConfigurable {
     private var includeGitMojiDescriptionConfig: Boolean = false
     private val textAfterUnicodeOptions = arrayOf("<nothing>", "<space>", ":", "(", "_", "[", "-")
     private val textAfterUnicode = ComboBox(textAfterUnicodeOptions)
+    private val languageOptions = arrayOf("auto", "en_US", "zh_CN")
```

Comment:
Hello, this is my first time developing an "idea" plugin and also my first time using Kotlin for development.

I feel that the plugin you've developed is very useful, so I'd like to add Chinese translation to the plugin for use.

In other words, I don't have enough experience in "idea" plugin development, and I may not know how to move this to class GitmojiLocale because I don't know how the plugin's UI works.

So I have a question, if I move it, how can I add it to the options in the configuration panel?

Should I use static methods? If possible, could you tell me a better way?

Reply:
I'm wondering if we should also maintain this collection online?

Reply:
That I propose is just tu add the list has a static property in the GitMojiLocale